platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -93,6 +105,13 @@ struct odp_crypto_global_s {
 
 static odp_crypto_global_t *global;
 
+typedef struct crypto_local_t {
+	uint8_t input[_ARM_CRYPTO_MAX_DATA_LENGTH];
```

Comment:
Since the buffer is now sometimes used for only output and sometimes for both input and output, it is misleading to name it "input".

Reply:
I will rename it to "data" or even simply "buffer" so that its purpose is clear. Thanks!